client/src/pages/Main/useStyles.ts

```diff
@@ -0,0 +1,36 @@
+import { makeStyles } from '@material-ui/core/styles';
+
+const useStyles = makeStyles(() => ({
+  root: {
+    height: '100vh',
+    backgroundColor: 'white',
+  },
+  mainContent: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  subBox: {
+    padding: '30px 15%',
```

Comment:
be careful with using percentages cause it'll all depend on its parents element width 

this is okay if you've tested all screensizes

Reply:
changed to padding: 30px 120px